docs/reference/commandline/run.md

```diff
@@ -253,6 +254,15 @@ Docker daemon.
 
 For in-depth information about volumes, refer to [manage data in containers](https://docs.docker.com/engine/tutorials/dockervolumes/)
 
+### Mount using the new interface (--mount)
+
+The new mount interface is described in the [service create command section](service_create.md).
+
+Examples:
+
+    $ docker run --read-only --mount type=volume,target=/icanwrite busybox touch /icanwrite/here
+    $ docker run -t -i --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock busybox sh
```

Comment:
We should use a different example than `/var/run/docker.sock` since ultimately we want to provide a different way of providing API access to the container.